tests/integration/ps.bats

```diff
@@ -3,18 +3,21 @@
 load helpers
 
 function setup() {
+	# ps requires cgroups
+	[ $EUID -ne 0 ] && requires rootless_cgroup
+
 	setup_busybox
+	if [ $EUID -ne 0 ]; then
+		# Rootless does not have default cgroup path.
+		set_cgroups_path
+	fi
```

Comment:
Maybe merge the two EUID checks into a single `if` block

Reply:
See, the first one is done before `setup_busybox` (which allows us to skip a bunch of actions that are wasted because the test is going to be skipped), and the second one is done after (because it requires config.json to be present).